CHANGELOG.md

```diff
@@ -80,6 +88,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Changed
 
+- Expose `ValidatorHashNotFound` constructor of `MkUnbalancedTxError`. ([#1545](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1545)
```

Comment:
I would remove this item entirely: unlikely it affects anyone, and yet it consumes reader's time